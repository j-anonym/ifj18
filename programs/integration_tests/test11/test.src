if(4==4) then
print "pravda"
else
print "nepravda"
end


if(5==5) then
print "\ndruha pravda"
end



if(4==5) then
print "pravda"
elsif(5==6) then
print "\npravda v elsif"
elsif(7==9) then
print "\npravda v elsif2"
else
print "\nvsade nepravdy\n"
end


