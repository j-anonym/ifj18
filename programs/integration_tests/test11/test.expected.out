pravda
druha pravda
vsade nepravdy
